src/main/kotlin/lotto/domain/Lotto.kt

```diff
@@ -0,0 +1,38 @@
+package lotto.domain
+
+import lotto.exceptions.LottoException
+import lotto.util.LottoUtils
+
+/***
+ * This Class represents a lotto ticket,
+ *  - contains 6 unique numbers
+ *  - in range 1 to 45
+ */
+class Lotto(val numbers: List<Int> = emptyList<Int>()) {
```

Comment:
```suggestion
class Lotto(val numbers: List<Int> = emptyList()) {
```
You can remove explicit type argument here.
But how is that possible? It would be great if you could look into this and tell me about it :)

**Hint**: type inference, generic function

Reply:
Yes, totally. Kotlin infer the type, emptyList() is a generic function that infer the type depending the context. So yes your are right, that's very redundant.